docs/modules/agents/toolkits/examples/sql_database.ipynb

```diff
@@ -501,13 +514,312 @@
    "source": [
     "agent_executor.run(\"Who are the top 3 best selling artists?\")"
    ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "8feee2be",
+   "metadata": {},
+   "source": [
+    "# Memory Use Case Example\n",
```

Comment:
this should be `##` or it will show up in the sidebar

Reply:
nah, i see. fixing now!